diff --git a/src/lib/ggpo/network/udp_proto.cpp b/src/lib/ggpo/network/udp_proto.cpp
index 8fdfa11..3e53bb4 100644
--- a/src/lib/ggpo/network/udp_proto.cpp
+++ b/src/lib/ggpo/network/udp_proto.cpp
@@ -16,7 +16,7 @@ static const int SYNC_RETRY_INTERVAL = 2000;
 static const int SYNC_FIRST_RETRY_INTERVAL = 500;
 static const int RUNNING_RETRY_INTERVAL = 200;
 static const int KEEP_ALIVE_INTERVAL    = 200;
-static const int QUALITY_REPORT_INTERVAL = 1000;
+static const int QUALITY_REPORT_INTERVAL = 100;
 static const int NETWORK_STATS_INTERVAL  = 1000;
 static const int UDP_SHUTDOWN_TIMER = 5000;
 static const int MAX_SEQ_DISTANCE = (1 << 15);
